kolla/node_custom_config/blazar.conf

```diff
@@ -7,7 +7,18 @@ exempt_projects = {% for project_id in blazar_usage_project_exemptions %}{{ proj
 
 {% if enable_blazar_allocation_enforcement | bool %}
 enabled_filters = MaxReservationLengthFilter, ExternalServiceFilter
+{% if blazar_external_service_endpoint is defined %}
```

Comment:
I think we should also default this to some values in the defaults.yml, b/c most sites are not going to deviate.

```yaml
blazar_external_service_check_create_endpoint: "{{ chameleon_portal_url }}/balance/check-create/"
... etc
```